### PR TITLE
Connected to Mongo database

### PR DESCRIPTION
**Pull request by KilgoreTrout9**
Connected to shared Mongo database on Atlas

Moved Data.jsx to root directory

### CODE REVIEW

**Review by mingwencode — COMMENTED** [review]
Do you wanna delete the package-lock in this PR? otherwise, lgtm

File: package.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
     "@react-google-maps/api": "^2.1.1",
     "axios": "^0.21.1",
     "babel-loader": "^8.2.2",
-    "body-parser": "^1.19.0",
     "bootstrap": "^4.6.0",
+    "connect-mongo": "^4.4.1",
```
Comment (greta-schock) [inline package.json:46]: When this is merged, please send out a reminder to everyone that we need to do another "npm install"

**Review by greta-schock — CHANGES_REQUESTED** [review]
Please remove package-lock.json from the PR

File: package.json (hunk 1 above)
Comment (KilgoreTrout9) [reply package.json:46]: Has been deleted

**Review by bryancWA — APPROVED** [review]

**Pull request merged**